tests/testthat/test-test.r

```diff
@@ -7,6 +7,8 @@ test_that("Package can be tested with testthat not on search path", {
     on.exit(attach(testthat_env, testthat_pos), add = TRUE)
   }
 
-  test("testTest")
-  test("testTestWithDepends")
+  test("testTest", reporter = "stop")
+  expect_true(TRUE)
```

Comment:
@hadley Is there anything nicer than this? For some reason, `expect_true(!is.null(test("...", ...)))` does not seem to work in this particular case.